### PR TITLE
Phishing url prediction

### PR DESCRIPTION
**Pull request by tomer-pan**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/45322

## Description
Adding the Phishing URL script to the main v3 playbook.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [x] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md [hunk 1]
```diff
@@ -99,11 +100,13 @@ This playbook does not use any integrations.
...
+| DBotPredictURLPhishing | When "True" the playbook will run the machine learning based "DBotPredictURLPhishing" automation \(available when "Phishing URL" pack is installed\).<br/>This automation will run several checks in order to determine the score of the URLs found in the email and will set a verdict for URLs which were found as "Suspicious" or "Malicious" and will add these URLs as indicators. Based on the verdict, the incident severity will be set \(Medium severity for "Suspicious" and High severity for "Malicious"\).<br/>The default number of extracted URLs is 3. In order to set a different number, please set the "DBotPredictURLPhishingURLsNumber" input.<br/>Please note:<br/>- False/True positives are possible.<br/>- This automation may take a few minutes.</br>- In order to get more accurate results you may want to install and enable the "Whois" pack (optional). | False | Optional |
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md:103]: Maybe make the whois pack clickable for the pack in marketplace

File: Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md [hunk 2]
```diff
@@ -99,11 +100,13 @@ This playbook does not use any integrations.
...
+| DBotPredictURLPhishingURLsNumber | The number of URLs to be extracted from the email html which will be analyzed in the "DBotPredictURLPhishing" automation. | 3 | Optional |
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md:104]: Add in the description about the default value

File: Packs/Phishing/ReleaseNotes/3_0_3.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Playbooks
+##### Phishing - Generic v3
+- `DBotPredictURLPhishing` automation was added to the ML branch.
+- Added 2 playbook inputs related to the `DBotPredictURLPhishing` automation.
```
Comment (David-BMS) [inline Packs/Phishing/ReleaseNotes/3_0_3.md:5]: Consider adding the names of the inputs.

**Review by David-BMS — APPROVED** [review]
Please review comments
@idovandijk please have a look too

File: Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md (hunk 1 above)
Comment (tomer-pan) [reply Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md]: Done

File: Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md (hunk 2 above)
Comment (tomer-pan) [reply Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md]: Done

File: Packs/Phishing/ReleaseNotes/3_0_3.md (hunk 3 above)
Comment (tomer-pan) [reply Packs/Phishing/ReleaseNotes/3_0_3.md]: Done

File: Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md [hunk 4]
```diff
@@ -99,11 +100,13 @@ This playbook does not use any integrations.
...
+| DBotPredictURLPhishing | When "True" the playbook will run the machine learning based "DBotPredictURLPhishing" automation \(available when "Phishing URL" pack is installed\).<br/>This automation will run several checks in order to determine the score of the URLs found in the email and will set a verdict for URLs which were found as "Suspicious" or "Malicious" and will add these URLs as indicators. Based on the verdict, the incident severity will be set \(Medium severity for "Suspicious" and High severity for "Malicious"\).<br/>The default number of extracted URLs is 3. In order to set a different number, please set the "DBotPredictURLPhishingURLsNumber" input. Default value is "False".<br/>Please note:<br/>- False/True positives are possible.<br/>- This automation may take a few minutes.</br>- In order to get more accurate results you may want to install and enable the [Whois pack](https://xsoar.pan.dev/marketplace/details/Whois) (optional). | False | Optional |
```
Comment (yaakovi) [inline Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md:103]:
```suggestion
| DBotPredictURLPhishing | When "True" the playbook will run the machine learning based "DBotPredictURLPhishing" automation \(available when "Phishing URL" pack is installed\).<br/>This automation will run several checks in order to determine the score of the URLs found in the email and will set a verdict for URLs which were found as "Suspicious" or "Malicious" and will add these URLs as indicators. Based on the verdict, the incident severity will be set \(Medium severity for "Suspicious" and High severity for "Malicious"\).<br/>The default number of extracted URLs is 3. In order to set a different number, please set the "DBotPredictURLPhishingURLsNumber" input. Default value is "False".<br/>Please note:<br/>- False/True positives are possible.<br/>- This automation may take a few minutes.<br/>- In order to get more accurate results, you may want to install and enable the [Whois pack](https://xsoar.pan.dev/marketplace/details/Whois) (optional). | False | Optional |
```

**Review by idovandijk — APPROVED** [review]

**Comment by julieschwartz18** [comment]
@tomer-pan - Doc review complete (see the commits for comments)

**Pull request merged**